lib/widgets/sign_in_widget.dart

```diff
@@ -10,52 +11,57 @@ import '../configuration/app_text.dart';
 import '../configuration/app_path.dart';
 import '../configuration/app_colors.dart';
 
-class SignIn extends StatefulWidget {
+class SignInWidget extends StatefulWidget {
   final Function _toggle;
 
-  SignIn({
+  SignInWidget({
     @required Function toggle,
   }) : _toggle = toggle;
 
   @override
-  _SignInState createState() => _SignInState();
+  _SignInWidgetState createState() => _SignInWidgetState();
 }
 
-class _SignInState extends State<SignIn> {
-  final formKey = GlobalKey<FormState>();
-  AuthMethods authMethods = AuthMethods();
-  DatabaseMethods databaseMethods = DatabaseMethods();
-  TextEditingController emailTextEditingController = TextEditingController();
-  TextEditingController passwordTextEditingController = TextEditingController();
-
+class _SignInWidgetState extends State<SignInWidget> {
   bool isLoading = false;
+
   QuerySnapshot snapshotUserInfo;
 
+  AuthMethods _authMethods = AuthMethods();
+  DatabaseMethods _databaseMethods = DatabaseMethods();
+
+  final formKey = GlobalKey<FormState>();
+  final emailValidator = ValidationBuilder().email().maxLength(50).build();
```

Comment:
I appreciate that you used custom package what I've recommend :) And if it was not at guide you are using for this app it's way better